be/src/exprs/runtime_filter.cpp

```diff
@@ -351,6 +359,26 @@ class RuntimePredicateWrapper {
         return Status::OK();
     }
 
+    Status change_to_bloom_filter() {
+        if (_filter_type != RuntimeFilterType::IN_OR_BLOOM_FILTER) {
```

Comment:
Looks like there is no where to check this Status?
And better not using `stringstream` in hot path.

Reply:
already change to return void and check by CHECK